docs/arf.md

```diff
@@ -1935,6 +1986,9 @@ More detailed requirements for the issuance process of PIDs and
 attestations, for instance regarding the issuance protocol, are included
 in \[Topic 10\] and \[Topic 23\].
 
+[**Can you add detail about how repeat issuances of attestations are handled. For example, I am issued with an attestation that allows me entry to a secure workplace for 1 week. This attestation has to be re-issued every week while I am working there. How can the issuer "push" a new attestation out to my wallet? What capabilities exist for issuer-initiated transactions? 
+Similarly for relying parties who may need to re-request information on a daily basis. Without the user having to initiate the action (eg by scanning a QR code), what capability does a relying party have to initiate a transaction?**
```

Comment:
Re-issuance of attestations will be covered in ARF 1.8, to be published Q1 2025. It is actively being discussed now.